feature/client/src/main/java/com/mifos/feature/client/syncClientDialog/SyncClientsDialogScreen.kt

```diff
@@ -27,35 +38,40 @@ import androidx.compose.ui.unit.dp
 import androidx.hilt.navigation.compose.hiltViewModel
 import androidx.lifecycle.compose.collectAsStateWithLifecycle
 import com.mifos.core.designsystem.component.MifosCircularProgress
+import com.mifos.core.objects.client.Client
 import com.mifos.feature.client.R
 
 @Composable
-fun SyncClientsDialogScreen(
-    viewModel: SyncClientsDialogViewModel = hiltViewModel(),
+internal fun SyncClientsDialogScreen(
     dismiss: () -> Unit,
     hide: () -> Unit,
+    list: List<Client>? = listOf(),
+    viewModel: SyncClientsDialogViewModel = hiltViewModel(),
 ) {
     val uiState by viewModel.syncClientsDialogUiState.collectAsStateWithLifecycle()
     val uiData by viewModel.syncClientData.collectAsStateWithLifecycle()
 
     LaunchedEffect(key1 = Unit) {
         viewModel.syncClient()
+        list?.let {
+            viewModel.setClientList(clientsList = list)
+        }
```

Comment:
Just a simple doubt. Shouldn't we set `clientListFirst` and `sync` later? 
![image](https://github.com/user-attachments/assets/546115e0-7161-47a8-878b-44307f886252)

Reply:
i'll check once. It most probably is

Reply:
you can take ref from `syncCenterDialogScreen` or you can check the old fragment itself

Reply:
Yeah, will do!!